src/components/Profile/ProfileInfo/ProfileStatus/ProfileStatusWithHook.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+
+import s from "./ProfileStatus.module.css";
+
+const ProfileStatusWithHooks = (props) => {
+  let [editMode, setEditMode] = useState(false);
```

Comment:
Consider using "const" instead of "let" for variables that do not change in the current context

Reply:
Done